.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
+# The following paragraph was taken from the Apache Spark project.
+#   see: https://github.com/apache/spark/blob/master/.github/autolabeler.yml
+# Bot page: https://github.com/apps/probot-autolabeler
+# The matching patterns follow the .gitignore spec.
+# See: https://git-scm.com/docs/gitignore#_pattern_format
+# Also, note that the plugin uses 'ignore' package. See also
+# https://github.com/kaelzhang/node-ignore
+
+INFRA:
+  - ".baseline/"
+  - ".github/"
+  - "/dev/"
+  - ".asf.yaml"
+  - ".gitattributes"
+  - ".gitignore"
+  - "baseline.gradle"
+  - "deploy.gradle"
+  - "jitpack.yml"
+  - "site/mkdocs.yml"
+  - "travis.yml"
+BUILD:
+  - "/dev/"
+  - "/project/"
+  - "/assembly/"
+  - "*pom.xml"
+  - "/bin/docker-image-tool.sh"
+  - "/bin/find-spark-home*"
+  - "gradle/"
+  - "**.gradle"
+BASH:
+  - "dev/check-license"
+  - "**.sh"
+DOCS:
+  - "site/"
+  - "!/site/mkdocs.yml"
```

Comment:
Does this exclude mkdocs.yml?

Reply:
Yes this excludes that. I believe I put it under INFRASTRUCTURE instead. However, I'm fully open to whatever you prefer as I'm very new to the project and don't spend a fraction of the amount of time looking at PRs that you do.

Reply:
Let's put it in docs, since it controls docs layout.

Reply:
Good call. The spark project, where I took my inspiration, has much more granular additions and usages of excludes. I prefer your approach to a simpler layout based on what things affect. 👍